### PR TITLE
Adds event collection to lenovo provider

### PR DESCRIPTION
**Pull request by rodneyhbrown7**
@miq-bot add_label: wip, providers/physical-infrastructure

### CODE REVIEW

**Comment by juliancheal** [comment]
@miq-bot add_label wip, providers/physical-infrastructure

File: app/models/manageiq/providers/lenovo/physical_infra_manager/event_catcher/stream.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    yield get_events.collect { |e| ManageIQ::Providers::Lenovo::PhysicalInfraManager::EventParser.event_to_hash(e, @ems.id) }
+  end
+
+  def event_monitor_handle
+    @event_monitor_handle ||= create_event_monitor_handle @ems
+  end
+
+  private
+
+  def get_events
```
Comment (juliancheal) [inline app/models/manageiq/providers/lenovo/physical_infra_manager/event_catcher/stream.rb]: We should rename this to `events`

**Comment by juliancheal** [comment]
This LGTM apart from renaming `get_event` to `events`. There was also some discussion over on the ManageIQ channel about using the Lonely Operator `&.` https://gitter.im/ManageIQ/manageiq?at=58b99fb021d548df2c8797b2

I think the use of it here is fine, but I'd like a second opinion @chrisarcand.

**Comment by chrisarcand** [comment]
The use of it here is correct.

**Pull request merged**